static/snippet/_LOG.md

```diff
@@ -0,0 +1,13 @@
+Genesis provides a class called `LOG` that can be used to insert custom log messages. This class provides you with 
+5 methods to be used accordingly: `info`, `warn`, `error`,`trade`,`debug`. To use these methods, you need to provide, as an input,
+a string that will be inserted into the Logs.
```

Comment:
@GuiWar I think we should explain that this LOG instance is a Logger from SLF4J and more documentation can be found in: https://www.slf4j.org/ .

Reply:
Sure!